cirq/api/google/v2/program.proto

```diff
@@ -146,10 +144,16 @@ message Arg {
 message ArgValue {
   oneof arg_value {
     float float_value = 1;
-    bytes bytes_value = 2;
+    RepeatedBoolean bool_values = 2;
```

Comment:
The more I stare at this, the more I think this should be 
RepeatedArgValue, which includes repeated versions of the arg values (not ArgValues, but things like repeated float_values).    I'd be tempted to use this in the Arg oneof.

Reply:
Ah that doesn't really help, since you still want to have the base recorded data be a repeat of a raw type.